test/pylib/scylla_cluster.py

```diff
@@ -1323,6 +1353,13 @@ async def _server_change_ip(self, request: aiohttp.web.Request) -> dict[str, obj
         ip_addr = await self.cluster.change_ip(server_id)
         return {"ip_addr": ip_addr}
 
+    async def _server_change_rpc_address(self, request: aiohttp.web.Request) -> dict[str, object]:
+        """Pass change_ip command for the given server to the cluster"""
+        assert self.cluster
+        server_id = ServerNum(int(request.match_info["server_id"]))
+        ip_addr = await self.cluster.change_rpc_address(server_id)
+        return {"ip_addr": ip_addr}
```

Comment:
This is confusing since the api is about changing rpc_address but it returns something calls `ip_addr`.
It would have been fine if it wasn't an existing loaded term.
Why not call the returned address `rpc_address` ?

Reply:
changed